lib/internal/loader/Loader.js

```diff
@@ -134,24 +104,16 @@ class Loader {
     }
 
     let inspectBrk = false;
-    if (this.isMain) {
-      if (process._breakFirstLine) {
-        delete process._breakFirstLine;
-        inspectBrk = true;
-      }
-      this.isMain = false;
+    if (process._breakFirstLine) {
```

Comment:
Reading this again, can you explain why removing the `if (this.main)` without `if (parentURL)` is valid?

Reply:
Because we clear `_breakFirstLine` it's guaranteed to only trigger for the first module loaded.